tools/solana/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM ubuntu:20.04
+WORKDIR /solana
+
+ADD https://github.com/solana-labs/solana/releases/download/v1.16.23/solana-release-x86_64-unknown-linux-gnu.tar.bz2 .
+RUN tar jxf solana-release-x86_64-unknown-linux-gnu.tar.bz2
+ENV PATH=/solana/solana-release/bin:$PATH
+
+ENTRYPOINT [ "solana-test-validator" ]
```

Comment:
just curious, would you also want prefunding the CL node?

you should be able to do that with the `solana airdop` command